src/Microsoft.Extensions.Primitives/project.json

```diff
@@ -17,17 +17,11 @@
   },
   "frameworks": {
     "net451": { },
-    "netcore50": {
-      "dependencies": {
-        "System.Resources.ResourceManager": "4.0.0",
-        "System.Runtime": "4.0.20",
-        "System.Threading": "4.0.10"
-      }
-    },
     "dotnet5.4": {
       "dependencies": {
-        "System.Runtime": "4.0.21-*",
-        "System.Resources.ResourceManager": "4.0.1-*"
+        "System.Diagnostics.Debug": "4.0.10",
```

Comment:
I'm working on that one.  See https://github.com/dotnet/corefx/issues/5707.  Once this is done you'll be able to reference latest for most packages.  There will still be a set that you shouldn't since it would bring in API that is not implemented by UWP RTM and it would break you out of shared library.  I'll doc what those are.